synfig-studio/src/gui/docks/dock_info.h

```diff
@@ -63,6 +67,7 @@ class Dock_Info : public Dock_CanvasSpecific
 
 	//! Current render progress - 0.0 to 1.0
 	//  depends on n_passes_requested and current_pass
+	void set_async_render(etl::handle<AsyncRenderer> *ar);
```

Comment:
you don't have to pass it as pointer (of pointer). Pass by value or reference.

Reply:
Yup!